ui/apps/platform/src/Components/ScannerV4IntegrationBanner.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Banner, Flex, FlexItem } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
+
+import useRestQuery from 'hooks/useRestQuery';
+import useMetadata from 'hooks/useMetadata';
+import { fetchImageIntegrations } from 'services/ImageIntegrationsService';
+import { getVersionedDocs } from 'utils/versioning';
+import { scannerV4ImageIntegrationType } from 'types/imageIntegration.proto';
+
+function ScannerV4IntegrationBanner() {
+    const { version } = useMetadata();
+    const { data, loading, error } = useRestQuery(fetchImageIntegrations);
+
+    // Don't show the banner if we don't have successful responses
+    if (!data || loading || error || !version) {
+        return null;
+    }
+
+    const hasScannerV4Integration = data.some(({ type }) => type === scannerV4ImageIntegrationType);
+
+    // Don't show the banner if we have a Scanner V4 integration
+    if (hasScannerV4Integration) {
+        return null;
+    }
+
+    return (
+        <Banner variant="info">
+            <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                justifyContent={{ default: 'justifyContentCenter' }}
+            >
+                <FlexItem>
+                    <InfoCircleIcon />
```

Comment:
No other banner, and especially relevant the other `variant="info"` banner, has an icon:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/MainPage/Banners/AnnouncementBanner.tsx#L37-L50